### ISSUE TITLE
ProtonMail Bridge compability

### ISSUE THREAD

**Issue by ghost** [nylas/nylas-mail#3645]
I don't know if this is the right place to post this. But I am asking if you will ever add support for [ProtonMail Bridge](https://protonmail.com/bridge/) that is in open beta now?

**Comment by vladimiry** [comment]
As I understood it, that Bridge thing is a desktop application (I guess built with Electron) that acts as an advanced proxy server running on the 127.0.0.0 host, which exposes the Protonmail's accounts via the standard IMAP/SMTP protocols. If that's so, then it should be possible to just add Protonmail's accounts to any mail client, see here https://protonmail.com/bridge/clients

**Comment by ghost** [comment]
So, the Bridge thing use default ports 1143 for incoming and 1025 for outgoing traffic. Nylas allows currently in the Windows version, port 143 or 993 for incoming and 25, 465, 587 for outgoing traffic when setting up an account via IMAP/SMTP. Under settings in the Bridge for ProtonMail, you can change IMAP and SMTP ports when doing so to adjust to Nylas ports (for example 143 and 25), I appear to get the following error: `connect ECONNREFUSED 127.0.0.1:25`

**Comment by vladimiry** [comment]
Does the Bridge work with a traditional desktop email clients (like Thunderbird, Claws Mail, Outlook, etc) as described here https://protonmail.com/bridge/clients? If it does not, then issue is probably not the Nylas specific.

**Comment by ghost** [comment]
So ProtonMail Bridge is currently in open beta. I have contacted them about supporting Nylas Mail in the future. I get it set up in Thunderbird, although you need to do security exceptions for the certificate of [IMAP](https://protonmail.com/bridge/thunderbird#6) and [SMTP](https://protonmail.com/bridge/thunderbird#11).

**Comment by siucdude** [comment]
Hi Everyone,
I use ProtonMail for work and personal usage.. Also our users use Linux, Mac & Windows... We can load the bridge on all 3 and it works great with Thunderbird, Evolution, MacMail, Outlook, but the one that all of us would love to use is for this client.

Thanks

**Comment by BeastOfBodmin** [comment]
> Does the Bridge work with a traditional desktop email clients (like Thunderbird, Claws Mail, Outlook, etc) as described here https://protonmail.com/bridge/clients? If it does not, then issue is probably not the Nylas specific.

Yes it does. Specifically, I can attest to it working with Postbox on macOS. Postbox is based on Thunderbird. The Protonmail Bridge setup instructions also mention Thunderbird specifically.
